app/models/bot/smooch.rb

```diff
@@ -542,7 +543,11 @@ def self.process_menu_option_value(value, option, message, language, workflow, a
   end
 
   def self.is_a_shortcut_for_submission?(state, message)
-    self.is_v2? && (state == 'main' || state == 'waiting_for_message') && (!message['mediaUrl'].blank? || ::Bot::Alegre.get_number_of_words(message['text'].to_s) > CheckConfig.get('min_number_of_words_for_tipline_submit_shortcut', 10, :integer))
+    self.is_v2? && (state == 'main' || state == 'waiting_for_message') && (
+      !message['mediaUrl'].blank? ||
+      ::Bot::Alegre.get_number_of_words(message['text'].to_s) > CheckConfig.get('min_number_of_words_for_tipline_submit_shortcut', 10, :integer) ||
+      URI.regexp.match(message['text'].to_s)!=nil # URL in message?
```

Comment:
it's better to use !URI.regexp.match(message['text'].to_s).nil?

Reply:
Amazing. I've made that change. Thank you!